src/util/config.cpp

```diff
@@ -677,7 +677,7 @@ configt::ansi_ct::c_standardt configt::ansi_ct::default_c_standard()
   #if defined(__APPLE__)
   // By default, clang on the Mac builds C code in GNU C11
   return c_standardt::C11;
-  #elif defined(__FreeBSD__)
+  #elif defined(__FreeBSD__) || defined(__OpenBSD__)
   // By default, clang on FreeBSD builds C code in GNU C99
   return c_standardt::C99;
```

Comment:
Please tweak the comment to say "clang on FreeBSD and OpenBSD".

Reply:
Also, the changes to c_preprocess.cpp suggest that the default isn't GNU C99 but actually proper standard C99?

Reply:
Yes.  OpenBSD does not use the GNU extensions AFAIK.  Either way, -std=gnuxx aren't valid options on OpenBSD.

Reply:
Comment updated as requested.